src/Agent/NewRelic/Agent/Core/Wrapper/WrapperMap.cs

```diff
@@ -60,10 +60,7 @@ public WrapperMap(IEnumerable<IWrapper> wrappers, IDefaultWrapper defaultWrapper
                 Log.Error("No wrappers were loaded.  The agent will not behave as expected.");
             }
 
-            if (Log.IsFinestEnabled)
-            {
-                Log.Finest($"WrapperMap has NonDefaultWrappers: {string.Join(", ", _nonDefaultWrappers)}");
-            }
+            Log.Finest("WrapperMap has NonDefaultWrappers: {0}", string.Join(", ", _nonDefaultWrappers));
```

Comment:
The previous code only incurs the cost of the string.Join call and allocations if finest logging is enabled, but this version will always incur that cost.

Reply:
Reverted